scripts/stage-docs.sh

```diff
@@ -56,6 +53,8 @@ fi
 BLUE='\033[0;34m'
 NC='\033[0m' # No Color
 
+WEBSITE_URL="https://user:${GH_TOKEN}@${WEBSITE_REPO_URL}"
```

Comment:
Have the action do the concatenation and just pass in the full url into the script.  That way, the script continues to know nothing about the fact it is used from an action.